data/core/rootview.lua

```diff
@@ -798,14 +799,25 @@ function RootView:on_mouse_pressed(button, x, y, clicks)
 end
 
 
-function RootView:on_mouse_released(...)
+function RootView:on_mouse_released(button, x, y)
   if self.dragged_divider then
     self.dragged_divider = nil
   end
   if self.dragged_node then
+    local ox, oy = self:get_content_offset()
+    if x < ox or x > self.size.x or y < oy or y > self.size.y then
+      local tab = self.dragged_node[1].views[self.dragged_node[2]]
+      if tab:is(DocView) then
+        local filename = tab.doc.abs_filename or tab.doc.filename or "unsaved"
+        core.fork(filename)
+      else
+        core.log("tab splitting is not supported for this View")
+      end
+    end
+    system.capture_mouse(false)
```

Comment:
I think the normal cursor should be restored here.

Reply:
It is restored. `system.capture_mouse(false)` restores normal capturing.

Reply:
I mean something like `core.request_cursor("arrow")`.

Reply:
ah, I'll do that.